stl/inc/format

```diff
@@ -2934,15 +3030,54 @@ struct formatter<basic_string_view<_CharT, _Traits>, _CharT>
 
 template <class _CharT, class... _Args>
 struct _Basic_format_string {
+    static constexpr size_t _Num_args = sizeof...(_Args);
+
     basic_string_view<_CharT> _Str;
+    size_t _Arg_use_count[_Num_args > 0 ? _Num_args : 1]{};
+    size_t _Estimated_size    = 0;
+    bool _Is_estimation_exact = false;
 
     template <class _Ty>
         requires convertible_to<const _Ty&, basic_string_view<_CharT>>
     consteval _Basic_format_string(const _Ty& _Str_val) : _Str(_Str_val) {
         if (_Is_execution_charset_utf8()) {
+#ifndef __clang__ // TRANSITION, clang consteval bug (likely https://github.com/llvm/llvm-project/issues/52648)

```

Comment:
This can be cited as LLVM-52648. Their redirector `https://llvm.org/PR%s` (mentioned in [Custom Autolinks](https://github.com/microsoft/STL/wiki/Custom-Autolinks)) handles old Bugzilla and new-to-GitHub IDs, and this is a new-to-GitHub ID.